tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -102,6 +118,12 @@ string DebugString(const nvinfer1::DataType trt_dtype) {
   }
 }
 
+string DebugString(const TrtPrecisionMode mode){
+  string mode_str;
+  TrtPrecisionModeToName(mode, &mode_str);
```

Comment:
This routine returns a status which needs to be handled.

Reply:
I actually tried here. `TF_ASSERT_OK` is only defined in test mode. And `TF_RETURN_IF_ERROR` does not work in test mode. Do you have any idea how to handle that one for both cases ?

```cpp
string DebugString(const TrtPrecisionMode mode){
  string mode_str;
  TF_RETURN_IF_ERROR(TrtPrecisionModeToName(mode, &mode_str));
  return StrCat("TrtPrecisionMode::", mode_str);
}

// Gives this error:

In file included from ./tensorflow/core/lib/core/errors.h:19,
                 from ./tensorflow/core/framework/tensor_shape.h:23,
                 from ./tensorflow/compiler/tf2tensorrt/convert/utils.h:23,
                 from tensorflow/compiler/tf2tensorrt/convert/utils.cc:16:
tensorflow/compiler/tf2tensorrt/convert/utils.cc: In function ‘std::string tensorflow::tensorrt::DebugString(tensorflow::tensorrt::TrtPrecisionMode)’:
./tensorflow/core/platform/errors.h:76:49: error: could not convert ‘_status’ from ‘tensorflow::Status’ to ‘std::string’ {aka ‘std::basic_string<char>’}
   76 |     if (TF_PREDICT_FALSE(!_status.ok())) return _status; \
      |                                                 ^~~~~~~
      |                                                 |
      |                                                 tensorflow::Status
./tensorflow/core/platform/errors.h:76:49: note: in definition of macro ‘TF_RETURN_IF_ERROR’
   76 |     if (TF_PREDICT_FALSE(!_status.ok())) return _status; \
      |                                                 ^~~~~~~
```

Reply:
Oh right, I'll use `TF_CHECK_OK()`